arrayfire/statistics.py

```diff
@@ -108,3 +108,38 @@ def corrcoef(x, y):
     real = real.value
     imag = imag.value
     return real if imag == 0 else real + imag * 1j
+
+def topk(data, k, dim=0, order=TOPK.DEFAULT):
+    """
+    Return top k elements along a single dimension.
+
+    Parameters
+    ----------
+
+    data: af.Array
+          Input array to return k elements from.
+
+    k: scalar. default: 0
+       The number of elements to return from input array.
+
+    dim: optional: scalar. default: 0
+         The dimension along which the top k elements are extracted.
```

Comment:
Can you add the note about dim=0 support here as well. It is there in tests but not in the documentation.

Reply:
Done.